src/timebar/charts/vessel-events.js

```diff
@@ -296,16 +307,22 @@ VesselEvents.propTypes = {
       type: PropTypes.string,
     })
   ).isRequired,
+  selectedEventID: PropTypes.string,
   highlightedEventIDs: PropTypes.arrayOf(PropTypes.string),
   outerScale: PropTypes.func.isRequired,
+  onEventHighlighted: PropTypes.func,
+  onEventClick: PropTypes.func,
   outerWidth: PropTypes.number.isRequired,
   outerHeight: PropTypes.number.isRequired,
   graphHeight: PropTypes.number.isRequired,
   tooltipContainer: PropTypes.instanceOf(Element),
 }
 
 VesselEvents.defaultProps = {
-  highlightedEventIDs: null,
+  selectedEventID: null,
+  highlightedEventID: null,
```

Comment:
You seem to have changed `highlightedEventIDs` to `highlightedEventID` in defaultProps but not in PropTypes.
Additionally, I think this error should be caught by the linter - or is it already the case?

Reply:
Thanks for the catch! the linter didn't complain but will review the rules to ensure this won't happen again

Reply:
Thanks!